plugins/search/providers/elastic_search/lib/model/items/ESearchEntryItem.php

```diff
@@ -203,8 +205,7 @@ public function getSingleItemSearchQuery(&$entryQuery, $allowedSearchTypes, &$qu
 				KalturaLog::log("Undefined item type[" . $this->getItemType() . "]");
 		}
 
-		if ($this->getItemType() == ESearchItemType::EXACT_MATCH
-			&& in_array($this->getFieldName(), self::$ignoreDisplayInSearchFields)
+		if (in_array($this->getFieldName(), self::$ignoreDisplayInSearchFields)
```

Comment:
why you removed the exact match type condition?

Reply:
The item type can now also be 'RANGE', we want to retrieve recycled entries when using 'recycled_at'